usaspending_api/references/v2/views/city.py

```diff
@@ -65,26 +65,23 @@ def prepare_search_terms(request_data):
     return [es_sanitize(field).upper() if isinstance(field, str) else field for field in fields]
 
 
-def create_elasticsearch_query(return_fields, scope, search_text, country, state):
-    query_string = create_es_search("wildcard", scope, search_text, country, state)
+def create_elasticsearch_query(return_fields, scope, search_text, country, state, limit):
+    query_string = create_es_search(scope, search_text, country, state)
     query = {
         "_source": return_fields,
         "size": 0,
         "query": {
-            "bool": {
-                "must": query_string,
+            "constant_score": {
```

Comment:
The ES docs are a little unclear on this, but I'm not certain that this constant_score result is even being looked at by the aggregation part of the query, which is all we care about.  What I mean is, I don't think the aggregation cares that the results all have the same score, because its still organizing around doc count. Do you have an example where this is actually making a noticeable impact?

Reply:
@IsaacRay - I tried out a few finicky ways to get these queries to run only in the _filter context_, and this one was one of the few that worked: https://www.elastic.co/guide/en/elasticsearch/guide/current/combining-filters.html

However, I am just now trying it out with a `bool` instead of `constant_score` and it seems to work the same. I'll report back.

Reply:
@IsaacRay not a definitive source, but some evidence that they're the same and `bool` should be fine: 

I also tested it locally with `bool` and ran the profiler and performance was on par (maybe better?). Changing now.